packages/shorebird_cli/lib/src/cache.dart

```diff
@@ -153,7 +153,20 @@ abstract class CachedArtifact {
   File get file =>
       File(p.join(cache.getArtifactDirectory(fileName).path, fileName));
 
-  Future<bool> isUpToDate() async => file.existsSync();
+  Future<bool> isUpToDate() async {
```

Comment:
I might rename it "validate" since it really doesn't do an "update" check, but also doesn't matter. 🙃

Reply:
Renamed to `isValid`